scripts/build_utils.sh

```diff
@@ -946,6 +946,12 @@ build_debs() {
             egrep "*\.(changes|deb|ddeb|dsc|gz)$" | \
             egrep -v "(Packages|Sources|Contents)" | \
             $venv/chacractl binary ${chacra_flags} create ${chacra_endpoint}
+
+	# extract cephadm binary and push
+	dpkg-deb --fsys-tarfile release/${vers}/cephadm_${vers}*.deb | tar -x -f - --strip-components=3 ./usr/sbin/cephadm
+	echo cephadm | $venv/chacractl binary ${chacra_flags} create ${chacra_endpoint}
+	rm ./cephadm
+
```

Comment:
```suggestion
	          # extract cephadm binary and push
	          dpkg-deb --fsys-tarfile release/${vers}/cephadm_${vers}*.deb | tar -x -f - --strip-components=3 ./usr/sbin/cephadm
	          echo cephadm | $venv/chacractl binary ${chacra_flags} create ${chacra_endpoint}
	          rm ./cephadm

```

Reply:
Is this change suggesting anoteher level of indent for the added stanza?  That would be wrong, no?  It's intended to line up with the "# write json file" comment

Reply:
> Is this change suggesting another level of indent for the added stanza? That would be wrong, no? It's intended to line up with the "# write json file" comment

It was suggesting that, yes.  Weird, it doesn't get rendered that way.  Here's what I see.

![Screenshot at 2021-09-27 17-41-58](https://user-images.githubusercontent.com/11281208/134989532-a15963fb-af78-438b-afba-e8863098dc5d.png)

Reply:
tabs instead of spaces?